cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Genesis.hs

```diff
@@ -249,13 +247,13 @@ unCoin (Shelley.Coin c) = fromIntegral c
 -- it as milliseconds.
 configSlotDuration :: ShelleyGenesis TPraosStandardCrypto -> Word64
 configSlotDuration sg =
-  floor $ 1000 * Shelley.getSlotLength (Shelley.sgSlotLength sg)
+  floor $ 1000 * (Shelley.sgSlotLength sg)
```

Comment:
`Use Ouroboros.Consensus.BlockchainTime.WallClock.slotLengthToMillisec`

Reply:
ok, thanks

Reply:
There's also `mkSlotLength :: NominalDiffTime -> SlotLength` in that same module.

Reply:
done